src/test/scala/dxWDL/exec/TaskRunnerTest.scala

```diff
@@ -250,4 +250,18 @@ class TaskRunnerTest extends FlatSpec with Matchers {
     it should "handle missing optional files" taggedAs(EdgeTest) in {
         runTask("missing_optional_output_file")
     }
+
+    it should "retry docker at least 5 times" taggedAs(EdgeTest) in {
```

Comment:
This test is better done as a standalone WDL task. The general rule is to have short tests in the unit-test section, but this could take a significant amount of time.